invenio/modules/records/models.py

```diff
@@ -202,13 +208,17 @@ def bibxxx(cls):
 
 for idx in range(100):
     Bibxxx = "Bib{0:02d}x".format(idx)
-    globals()[Bibxxx] = type(Bibxxx, (db.Model, BibxxxMixin), {})
+    globals_Bibxxx = type(Bibxxx, (db.Model, BibxxxMixin), {})
```

Comment:
why did you split it into two lines? maybe can you just break the line inside brackets?